docs/development/manufacturer/manufacturer-design-guidelines.md

```diff
@@ -165,6 +166,53 @@ For connector pinout please refer to the [Betaflight Connector Standard](connect
 
 Selecting an appropriate IMU for flight controller operation is critical to the resulting flight performance of systems. Proven examples of hardware using single Invensense MPU-6000, single or dual ICM-20602, and also Bosch BMI-270 and BMI-180 units have been successfully demonstrated to operate with Betaflight, although the latter two examples have required significand development effort to bring performance of the IMU gyroscope and accelerometer sensing behaviors up to the standards required to maximize flight performance.
 
+:::note
```

Comment:
Admonitions won't work with .md extension?

Reply:
@VitroidFPV @freasy please look into the fact we are not able to commit when having set up local build with npm while changing files with mdx extension.

Have used two terminals on same PR in different folders - both with the changes wanting to submit - but only one with build setup for testing. Using first terminal to commit.

This seems to work as git clean -xdf does not remove everything after having build setup.

Reply:
Yes I have tried all manner of things and the only way I can commit a file with an .mdx extension is to block all checking.

Reply:
`:::note` `:::warning` and similar admonitions all work in files with an `.md` extension.  I think docusaurus handles them while creating the html, regardless of the extension.

Reply:
I think this is the right place, it is information for manufacturers.

Reply:
@ctzsnooze Did not verify 

@VitroidFPV @freasy but would it mean we don't have to use mdx extension at all? Simplifying contribution here?

Single document changes can be made easy with github functionality. (Maybe even have a button `edit this page on github` to stimulate contribution).

Reply:
I will have a fix PR up hopefully by today

Reply:
anything else to do? or is this approve-able as is?

Reply:
I think it can be approved and merged as-is.  There appears to be no functional difference in pages whether their extension is .md or .mdx.  We are hoping that @VitroidFPV can figure out why I can commit a change for .md files but not .mdx, that's all.